pom.xml

```diff
@@ -14,6 +14,13 @@
     <java.level>8</java.level>
   </properties>
 
+  <repositories>
```

Comment:
Typically we also add `pluginRepositories`. Not sure if it is needed in this case or not.